modules/indexing/src/test/java/org/apache/ignite/internal/mxbean/SqlQueryMXBeanImplSqlFreeMemQuotaExceededTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.mxbean;
+
+import org.apache.ignite.IgniteCache;
+import org.apache.ignite.cache.query.SqlFieldsQuery;
+import org.apache.ignite.cache.query.exceptions.SqlMemoryQuotaExceededException;
+import org.apache.ignite.internal.metric.UserQueriesTestBase;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.junit.After;
+import org.junit.Test;
+
+/**
+ * Tests SQL Free Memory works correctly after "global memory quota exceeded" event
+ *
+ */
+public class SqlQueryMXBeanImplSqlFreeMemQuotaExceededTest extends UserQueriesTestBase {
+    /**
+     * Teardown.
+     */
+    @After
+    public void stopAll() {
+        stopAllGrids();
+    }
+
```

Comment:
Two empty lines, Why the TCBot visa is green

Reply:
removed extra line